rubygem/lib/zeus/rails.rb

```diff
@@ -182,6 +182,14 @@ def test_environment
     end
 
     def test_helper
+      # don't let minitest setup another exit hook
+      begin
+        require 'minitest/unit'
+        MiniTest::Unit.class_variable_set("@@installed_at_exit", true)
```

Comment:
You're right, I didn't exactly ask a question.

I'd like to make zeus and minitest work out of the box without having to introduce hacks like this. Is there any way to invoke Minitest to run with specified options and then bypassing the autorun functionality? e.g.

```ruby
require 'test_helper'
class LameTest < ActiveSupport::TestCase
  def test_truth
    assert true
  end
end

Minitest.run(['-n', '/truth/'])
```
When run through rails, `Minitest.autorun` gets called when you require `rails/test_help` which causes the tests to get invoked twice. There don't appear to be any hooks to skip this and test_help requires and adds lots of useful code for Rails tests.

You wrote:
- why is running via autorun a bad thing?
- who is invoking autorun if zeus is running the tests directly?
- why not let the child process (I'm assuming there) exit, so it runs the tests normally?

Autorun doesn't work for this situation because we preload the test_helper which sets up autorun and then we can't specify our options to the actual `run` call in the `at_exit` hook. Would you be open to adding a method to customize the [run args](https://github.com/seattlerb/minitest/blob/master/lib/minitest.rb#L56) that are used in the `at_exit` hook?

As for your last point, that's what we're doing - the child process exits after we call `Minitest.run(args)` but then autorun runs them again with no arguments.